source/api/ut_runner.pkb

```diff
@@ -264,5 +264,39 @@ create or replace package body ut_runner is
     end loop;
   end;
 
+  function hash_suite_path(a_path varchar2, a_random_seed positiven) return varchar2 is
+    l_start_pos pls_integer := 1;
+    l_end_pos   pls_integer := 1;
+    l_result    varchar2(4000);
+    l_item      varchar2(4000);
+    l_at_end    boolean := false;
+  begin
+    if a_random_seed is null then
+      l_result := a_path;
+      end if;
+    if a_path is not null then
+      loop
+        l_end_pos := instr(a_path,'.',l_start_pos);
+        if l_end_pos = 0 then
+          l_end_pos := length(a_path)+1;
+          l_at_end  := true;
+          end if;
+        l_item := substr(a_path,l_start_pos,l_end_pos-l_start_pos);
+        if l_item is not null then
+          l_result  :=
+            l_result ||
+              dbms_crypto.hash(
```

Comment:
Would not be better to remove function `get_hash` from `ut_compound_data_helper` and put it into `ut_utils` then refer to that function rather than directly to  `dbms_crypto.hash`  that way it would be kept in one place?

Reply:
Ouch. Didn't notice the comment.
Can take care of it later.